tests/robot-tests/README.md

```diff
@@ -59,137 +58,139 @@ If you intend to run the tests from your local machine, you will also need to cr
 
-## Directory structure
 
+## **Directory structure**
 This section details what the various directories in robot-tests contain.
 
-### backup-data
-This directory holds backup data for both the MsSQL databases, and the emulated storage blobs and tables. If you run `backup-local.py`, the backup is stored here. If you run `restore-local.py`, it uses the data in this directory to restore to your docker databases and emulated Azure storage.
-
-### scripts
-This directory holds scripts used by `run\_tests.py` and the CI pipeline.
+## scripts
+This directory holds scripts used by `run_tests.py` and the CI pipeline.
 
-### test-results
+## test-results
 This directory holds the output of a test run, including the test report and log. Screenshots are preserved across runs, but other files are overwritten.
 
-### tests
+## tests
 This holds the actual robot framework/selenium tests. The tests are themselves organised into different folders. The `libs` doesn't contain tests, but utility keywords used by the tests. Similar to `libs`, `files` contains files used by the tests.
 
-### webdriver
+## webdriver
 This holds chromedriver, used by selenium to interact with the browser.
 
 
-## Guidelines for people writing UI tests
+## **Guidelines for people writing UI tests**
 
-### Parallelism / Pabot
-It is essential that the test suites can run in parallel. This might not be the case if one test suite relies on test data that another changes. AND this might cause the tests, when run in parallel, to fail in unpredictable ways, making it difficult to determine what test data is the failure-making culprit.
+## Parallelism / Pabot
+It is essential that the test suites can run in parallel. This might not be the case if one test suite relies on test data that another changes. This might cause the tests, when run in parallel, to fail in unpredictable ways, making it difficult to determine what test data is the failure-making culprit.
 
 For this reason, you **MUST** ensure that if a test suite changes test data, that you create test new data to be used specifically with that suite. Be careful if scavenging test data from other test suites! Ideally, every test suite will use test data that is only used by those particular tests.
 
-### Test data
-After a group discussion, it was decided that tests will, as far as is possible, create their own test data. This means that any tests you write that alter data, you will need to create that data from scratch.
-
-### IDE
+## Test data:
 
-If searching for an IDE to add/edit these tests, consider using IntelliJ with the following plugins:
+After a group discussion, it was decided that tests will, as far as is possible, create their own test data. This means that any tests you write that alter data, you will need to create that data from scratch.
```

Comment:
There is a set of tests where this isn't the case and they use that BootstrapData to set up various read-only Publications and Releases.  Hopefully when publishing is sped up, we can do away with this.